src/components/Tile.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+
+function Tile({image, title, children}) {
+    // if there is an image create the image tile
+    if (image) {
+        return (
+            <>
+                <section>
+                    <img src={image} alt={image.toString()}/>
+                </section>
+            </>
+        )
+    //when there is no image create a text-tile
+    //in the props.children you can make multiple elements
+    } else {
+        return (
+            <>
+                <section>
+                    <h2>{title}</h2>
+                    <p>{children}</p>
+                </section>
+            </>
+        );
+    }
+}
+
```

Comment:
Ik had zelf ook zoiets gedaan, ongeveer zoals jij, maar dan met een property type met waarde 'img' of 'text'. In de uitwerkingen wordt dit echter helemaal niet gebruikt, ze doen daar:
`function Tile({ img, imgDescription, title, children }) {
  return (
    <section>
      {img && <img src={img} alt={imgDescription}/>}
      <h2>{title}</h2>
      {children}
    </section>
  );
}
`
Ik begrijp het gedeelte met 'img &&' niet, jij wel?